datashader/spatial/viewshed.py

```diff
@@ -87,12 +53,7 @@
 
 # this value is returned by findMaxValueWithinDist() if there is no key within
 # that distance
-SMALLEST_GRAD = float('-inf')
-
-PROJ_LL = 0
-PROJ_NONE = -1
-
-NAN = -9999999999999999
+SMALLEST_GRAD = -9999999999999999999999.0
```

Comment:
What made this have to revert to an arbitrary constant? A switch from float to int?

Reply:
there were warning when placing this value `-inf` values in arrays. This may not be as pretty, but `float(-inf)` and `np.NINF` through invalid value warnings.

Reply:
the NAN constand is no longer being used

Reply:
This may or may not be related to your question, but the NOJIT tests revealed an issue where numba was impliciting casting floats to ints, when they should have been ints to begin with. a few days ago I fixed the NOJIT case by manually calling `int()` in some spots.  To be clear, numba was being helpful...but I want to avoid the additional casting so did some more refactoring.  the int NAN value was no longer need and also the `_is_null` wrapper around the int NAN constrant was removed.